great_expectations/checkpoint/checkpoint.py

```diff
@@ -115,7 +201,14 @@ def run(
         run_time=None,
         result_format=None,
         **kwargs,
-    ):
+    ) -> List[ValidationOperatorResult]:
+        runtime_configuration: dict = runtime_configuration or {}
+        result_format: Union[dict, None] = result_format or runtime_configuration.get(
```

Comment:
I wonder if we should just adopt Optional[dict) for variables that have "Union["some type", None] like this for now...

Reply:
Sounds good.